src/linter/ui5Types/amdTranspiler/pruneNode.ts

```diff
@@ -21,11 +21,13 @@ export class UnsafeNodeRemoval extends Error {
  */
 export default function (node: ts.Node) {
 	let nodeToRemove: ts.Node | undefined = node;
+	let greatestRemovableNode: ts.Node | undefined = undefined;
```

Comment:
@d3xter666 is this change needed?

Reply:
To bo honest, I really don't understand why this has been introduced. I checked the code and it seems redundant now. Maybe, it's just a leftover

Reply:
Done